db/queryUtils.js

```diff
@@ -5,20 +5,13 @@ exports.checkArticleExists = (article_id) => {
     .query(`SELECT * FROM articles
      WHERE article_id = $1;`, [article_id])
     .then((article) => {
```

Comment:
Would using selectArticleByArticleId not have the same effect? If yes, I probably wouldn't go and essentially create the same function again - we try to keep our coding DRY (which stands for **D**on't **R**epeat **Y**ourself
I don't think you are using this function for your patch, but just as a heads-up (: >

If you like the idea of using a utils function, it might be worth creating a utils function that is less specific - Dave had posted some notes in your seminar channel on Tuesday that you might find helpful looking at